monkey-runner/plugin/src/main/groovy/com/novoda/monkey/MonkeyConfigurationPlugin.groovy

```diff
@@ -33,30 +33,33 @@ public class MonkeyConfigurationPlugin implements Plugin<Project> {
     private void configureTask(Project project, MonkeyRunnerExtension extension) {
         def runMonkeyAllTask = project.task(TASK_NAME)
 
-        def android = project.extensions.findByName("android")
+        def android = project.android
         android.command.devices().eachWithIndex { device, index ->
 
             def monkeyTask = project.task("runMonkeyDevice${index}", type: TargetedMonkey) {
                 packageName = extension.packageNameFilter
-                events = extension.eventsCount
+                adb = android.command.adb
```

Comment:
So, this was discussed with @devisnik 
We want `AdbTask` to be self-contained. It should be able to work on its own. 
When the plugin create the tasks, it of course configures the tasks and this is not necessary. 

But here, we are creating a new task with the type of `TargetedMonkey`. This created task is out of reach of the plugin. 

Before the task itself was grabbing the plugin. 

In the future, we are planning to have more customizations so that creating tasks manually would not be necessary.